### PR TITLE
Grant aws-kf-ci-bot write access to kfserving

### PR DESCRIPTION
**Pull request by PatrickXYS**
Given the offline discussion with Serving WG TL:

@animeshsingh @yuzisun @ellistarn 

We have agreed to start migration work to Shared Test-infra for Kfserving.

### CODE REVIEW

File: github-orgs/kubeflow/org.yaml
Comment (ellistarn) [inline github-orgs/kubeflow/org.yaml:1008]: Why is this changing at the same time?

Comment (PatrickXYS) [reply github-orgs/kubeflow/org.yaml:1008]: Because there were a weird sync issue we notice before https://github.com/kubeflow/internal-acls/pull/362, we're still not sure if this is the root-cause, but it won't hurt

**Review by ellistarn — APPROVED** [review]

**Comment by Bobgy** [comment]
I think the original agreement is that we won't expand the number of repos until we've figured out a scalable solution for webhook configuration.

Have we reached that goal?

/cc @jlewi

**Comment by PatrickXYS** [comment]
/hold

For webhook management, there's a work-around. Will keep updated here

**Comment by jlewi** [comment]
@kubeflow/wg-serving-leads have you discussed creating your own GitHub org so that you could take on the admin costs of your repo/repo(s)?

**Comment by Bobgy** [comment]
This is minimal set up, so I'll do it.
Please keep in mind, there will always be delays for stuff like this, because I am a 20% maintainer for github org. Consider migrating to your own org if that's blocking your work as Jeremy suggests.

**Pull request merged**
